integration/spark/src/main/scala/org/apache/spark/sql/execution/command/mutation/CarbonProjectForDeleteCommand.scala

```diff
@@ -108,9 +108,6 @@ private[sql] case class CarbonProjectForDeleteCommand(
       }
       val executorErrors = ExecutionErrors(FailureCauses.NONE, "")
 
-      // handle the clean up of IUD.
-      CarbonUpdateUtil.cleanUpDeltaFiles(carbonTable, false)
```

Comment:
if we don't clean up stale delta files, it will be used after the next update.
maybe we can't remove it now.

Reply:
Yes, the purpose of PR is to prevent delete data by accident, after checking the code of `cleanUpDeltaFiles` function, I think it's safe to keep it here since it will only delete the files in the aborted scenario and it's crucial for data accuracy. I will revert removing this function.